pom.xml

```diff
@@ -252,6 +260,12 @@
                 <version>${geotools.version}</version>
                 <scope>${geotools.scope}</scope>
             </dependency>
+            <dependency>
+                <groupId>org.geotools</groupId>
```

Comment:
I am confused. Since we use ucar NetCDF reader, why use the reader from `geotools`?

Reply:
This is not needed, removed.